doc/source/code_ref/coordinates.rst

```diff
@@ -0,0 +1,208 @@
+SunPy coordinates
+=================
+
+The SunPy coordinates submodule is an implementation of the common solar physics
+coordinate frames using the :ref:`Astropy coordinates framework <astropy:astropy-coordinates>`.
+
+.. warning::
+
+   The accuracy of the transformations in this module have not been rigorously
+   verified. They have been compared to `sunpy.wcs` and match to numerical
+   precision. Independent verification will be added at a later date.
+
+
+Getting Started
+---------------
+
+The easiest interface to the coordinates module is through the `~astropy.coordinates.SkyCoord` class::
+
+  >>> import astropy.units as u
+  >>> from astropy.coordinates import SkyCoord
+  >>> import sunpy.coordinates
+  >>> c = SkyCoord(-100*u.arcsec, 500*u.arcsec, frame='helioprojective')
+  >>> c = SkyCoord(90*u.deg, -76*u.deg, frame='helioprojectiveradial')
+  >>> c = SkyCoord(x=-72241.0*u.km, y=361206.1*u.km, z=589951.4*u.km, frame='heliocentric')
+  >>> c = SkyCoord(70*u.deg, -30*u.deg, frame='heliographicstonyhurst')
+  >>> c
+  <SkyCoord (HelioGraphicStonyhurst: dateobs=None): (lon, lat, rad) in (deg, deg, km)
+      (70.0, -30.0, 695508.0)>
+
+
+SunPy implements support for the following solar physics coordinate systems:
+
+* Helioprojective (Cartesian) `~sunpy.coordinates.frames.HelioProjective`
+* Helioprojective (Radial) `~sunpy.coordinates.frames.HelioProjectiveRadial`
+* Heliocentric `~sunpy.coordinates.frames.HelioCentric`
+* Heliographic Stonyhurst `~sunpy.coordinates.frames.HelioGraphicStonyhurst`
+* Heliographic Carrington `~sunpy.coordinates.frames.HelioGraphicCarrington`
+
+for a complete description of these frames, see `sunpy.coordinates.frames`, for
+a more detailed description of the frames see [(Thompson 2006)](http://dx.doi.org/10.1051/0004-6361:20054262).
```

Comment:
I will add the contents of the gist to the sunpy example gallery at a later date.